src/fides/api/models/pre_approval_webhook.py

```diff
@@ -24,13 +33,26 @@ def persist_obj(
 
 
 class PreApprovalWebhookReply(Base):
+    """
+    Stores the callback response to the PreApprovalWebhook to determine if the Privacy
+    Request is eligible to be automatically approved
+    """
+
     webhook_id = Column(
-        String, ForeignKey(PreApprovalWebhook.id_field_path), nullable=False
+        String,
+        ForeignKey(PreApprovalWebhook.id_field_path, ondelete="SET NULL"), index=True
     )
```

Comment:
New: adding an index on this column (similar to the one on priavcy_request_id because it is commonly queried. Further, I'm setting this field to null in the event the webhook itself is deleted.

Reply:
yes good question, the main disadvantage is that it slows down writes, you don't want to/need to index everything.  This seemed like one of the fields that would be queried frequently and this will be a large table.